build/gulpfile.vscode.web.js

```diff
@@ -153,24 +153,26 @@ exports.createVSCodeWebFileContentMapper = createVSCodeWebFileContentMapper;
 
 const optimizeVSCodeWebTask = task.define('optimize-vscode-web', task.series(
 	util.rimraf('out-vscode-web'),
-	common.optimizeTask({
-		src: 'out-build',
-		entryPoints: _.flatten(vscodeWebEntryPoints),
-		otherSources: [],
-		resources: vscodeWebResources,
-		loaderConfig: common.loaderConfig(),
-		externalLoaderInfo: util.createExternalLoaderConfig(product.webEndpointUrl, commit, quality),
-		out: 'out-vscode-web',
-		inlineAmdImages: true,
-		bundleInfo: undefined,
-		fileContentMapper: createVSCodeWebFileContentMapper('.build/web/extensions', product)
-	})
+	optimize.optimizeTask(
```

Comment:
This call seems all broken. I would have expected again the format `{ out: '...', amd: { ... } }`

Reply:
https://github.com/microsoft/vscode/pull/160957/commits/bcbac86f7a521d4cd62ca86c2844e60f1acfe0f9